ddtrace/opentelemetry/example_test.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package opentelemetry_test
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel"
+
+	ddotel "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentelemetry"
+	ddtracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func Example() {
+	// Create a TracerProvider, optionally providing a set of options,
+	// that are specific to Datadog's APM product.
+	provider := ddotel.NewTracerProvider(ddtracer.WithService("opentelemetry_service"))
```

Comment:
Doing `WithProfiler` or something could be cool instead (specifically configure it with something that couldn't be done using the typical OTel API)